docs/channels/channel-form/fields.md

```diff
@@ -124,14 +124,54 @@ Simple relationship fields with only 1 relationship allowed are very similar in
       {/options:my_field_name}
     </select>
 
-For relationship fields that allow multiple selections, you may also set the sort order:
-
-    {options:my_field_name}
-        Order:
-        <input type="text" name="my_field_name[sort][]" value="{option_order}" />
-        <input type="checkbox" name="my_field_name[data][]" value="{option_value}" {checked}/>
-        {option_name}<br />
-    {/options:my_field_name}
+For relationship fields that allow multiple selections, you may also set the reorderable drag-and-drop lists:
+
+    {exp:channel:entries channel="yourChannel"}
+      <ul id="list-group">
+        {relationship_field}
+            <li class="list-item">
+                <div class="list-item__handle">::</div>
+                <div class="list-item__content">
+                    <div class="list-item__title">{relationship_field:title}</div>
+                    <div class="list-item__secondary"><span> #{relationship_field:entry_id} / </span>{relationship_field:status}</div>
+                </div>
+            </li>
+        {/relationship_field}
+      </ul>
+    {/exp:channel:entries}
+
+You need add SortableJs and a bit of Javascript code:
```

Comment:
Can we rephrase it, so people understand that SortableJS is optional?
Like "You would need need a bit of Javascript code (the example below is using SortableJs library):"

Also, screenshot here would look nice!